topi/python/topi/arm_cpu/conv2d_alter_op.py

```diff
@@ -59,6 +59,10 @@ def _alter_conv2d_layout(attrs, inputs, tinfos, out_type):
     data, kernel = tinfos
     out_dtype = out_type.dtype
 
+    # We only perform layout alteration for NCHW data layout.
+    if data_layout == "NHWC":
+        return None
```

Comment:
@anijain2305 I don't understand why we need this? If the data_layout is "NHWC", it should not use the topi template for "NCHW". So it won't perform layout alteration.